Project4.cpp

```diff
@@ -2,6 +2,7 @@
 #include <cmath>
 #include <string>
 #include <ctype.h>
+#include<stdlib.h>
```

Comment:
@xosantacruz can you add a space between the `include` and the `<stdlib.h>` please?